server/mailrepository/mailrepository-api/src/test/java/org/apache/james/mailrepository/MailRepositoryContract.java

```diff
@@ -206,19 +206,19 @@ default void shouldPreserveDsnParameters() throws Exception {
         Mail mail = createMail(MAIL_1);
         mail.setDsnParameters(dsnParameters);
 
-        testee.store(mail);
+        var key = testee.store(mail);
 
-        assertThat(testee.retrieve(MAIL_1).dsnParameters()).contains(dsnParameters);
+        assertThat(testee.retrieve(key).dsnParameters()).contains(dsnParameters);
     }
 
     @Test
     default void retrieveShouldReturnNullAfterRemoveAll() throws Exception {
         MailRepository testee = retrieveRepository();
-        testee.store(createMail(MAIL_1));
+        var key = testee.store(createMail(MAIL_1));
```

Comment:
I thought we once upon a time decided to not use `var` as it looses type explicitness.